### ISSUE TITLE
"py -3" command does not work when pyenv is used.

### ISSUE THREAD

**Issue by SosUts** [tlienart/Franklin.jl#1028]
Hi! Thank you for the great package!
I have one question related to windows user.

If windows user installed Python with pyenv, I think Python would be launched by "python" command.  
Therefore, "py -3" will cause an error in FD_HAS_PY3().
How about returning "python" when ENV["PYENV"] exists?

https://github.com/tlienart/Franklin.jl/blob/5c8f5e1af7d1279f5aad9e1437f7d883f4302307/src/build.jl#L1-L11

**Comment by tlienart** [comment]
Hello! 

Yes I think what you say makes sense. Maybe we could just add an `if` around line 6, would you be interested in a PR?

Note: the Python path in Franklin is bound to disappear. It is only used to run the minification (this script: https://github.com/tlienart/Franklin.jl/blob/master/src/scripts/minify.py relying on [css_html_js_minify](https://github.com/juancarlospaco/css-html-js-minify) which is unmaintained and not great but simple); it's actually not particularly helpful so the next planned major release gets rid of Python. 

In short: if this is bothering you and you'd like to have a small fix for it for now, I'll gladly merge a PR to this effect, I was just commenting on the fact that in the medium term this will all go away :)
